config/initializers/mapit.rb

```diff
@@ -0,0 +1 @@
+MAPIT_BASE_URL = "#{Plek.find('mapit')}/"
```

Comment:
Again, we're going to have more connections, so keeping this tidy is a good idea.